lib/DDG/Spice/Emojipedia.pm

```diff
@@ -0,0 +1,27 @@
+package DDG::Spice::Emojipedia;
+
+# ABSTRACT: Provide definitions and meanings for emoji characters.
+
+use strict;
+use utf8;
+use DDG::Spice;
+
+spice is_cached => 1;
+spice proxy_cache_valid => "200 1d";
+
+spice wrap_jsonp_callback => 1;
+
+spice to => 'http://api.emojipedia.org/emojis/$1?api_key={{ENV{DDG_SPICE_EMOJIPEDIA_APIKEY}}}';
+
+my $emoji_pattern = "[©®‼⁉™ℹ↔-↙↩-↪⌚-⌛⌨⏏⏩-⏳⏸-⏺Ⓜ▪-▫▶◀◻-◾☀-☄☎☑☔-☕☘☠☢-☣☦☪☮-☯☸-☺♈-♓♠♣♥-♦♨♻♿⚒-⚔⚖-⚗⚙⚛-⚜⚠-⚡⚪-⚫⚰-⚱⚽-⚾⛄-⛅⛈⛎-⛏⛑⛓-⛔⛩-⛪⛰-⛵⛷-⛸⛺⛽✂✅✈-✉✏✒✔✖✝✡✨✳-✴❄❇❌❎❓-❕❗❣-❤➕-➗➡➰➿⤴-⤵⬅-⬇⬛-⬜⭐⭕〰〽㊗㊙🀄🃏🅰-🅱🅾-🅿🆎🆑-🆚🈁-🈂🈚🈯🈲-🈺🉐-🉑🌀-🌡🌤-🎄🎆-🎓🎖-🎗🎙-🎛🎞-🏁🏅-🏆🏈-🏉🏌-🏰🏳-🏵🏷-👀👄-👅👑-👥👪-👭👹-👻👽-💀💄💈-💩💫-📽📿-🔽🕉-🕎🕐-🕧🕯-🕰🕳-🕴🕶-🕺🖇🖊-🖍🖤-🖥🖨🖱-🖲🖼🗂-🗄🗑-🗓🗜-🗞🗡🗣🗨🗯🗳🗺-🙄🙈-🙊🚀-🚢🚤-🚳🚷-🚿🛁-🛅🛋-🛒🛠-🛥🛩🛫-🛬🛰🛳-🛶🤐-🤗🤙-🤞🤠-🤧🤰🤳-🤾🥀-🥋🥐-🥞🦀-🦑🧀]|#⃣|0⃣|1⃣|2⃣|3⃣|4⃣|5⃣|6⃣|7⃣|8⃣|9⃣|☝[🏻-🏿]?|⛹[🏻-🏿]?|✊[🏻-🏿]?|✋[🏻-🏿]?|✌[🏻-🏿]?|✍[🏻-🏿]?|🇦[🇨-🇬🇮🇱-🇲🇴🇶-🇺🇼-🇽🇿]?|🇧[🇦-🇧🇩-🇯🇱-🇴🇶-🇹🇻-🇼🇾-🇿]?|🇨[🇦🇨-🇩🇫-🇮🇰-🇵🇷🇺-🇿]?|🇩[🇪🇬🇯-🇰🇲🇴🇿]?|🇪[🇦🇨🇪🇬-🇭🇷-🇺]?|🇫[🇮-🇰🇲🇴🇷]?|🇬[🇦-🇧🇩-🇮🇱-🇳🇵-🇺🇼🇾]?|🇭[🇰🇲-🇳🇷🇹-🇺]?|🇮[🇨-🇪🇱-🇴🇶-🇹]?|🇯[🇪🇲🇴-🇵]?|🇰[🇪🇬-🇮🇲-🇳🇵🇷🇼🇾-🇿]?|🇱[🇦-🇨🇮🇰🇷-🇻🇾]?|🇲[🇦🇨-🇭🇰-🇿]?|🇳[🇦🇨🇪-🇬🇮🇱🇴-🇵🇷🇺🇿]?|🇴🇲?|🇵[🇦🇪-🇭🇰-🇳🇷-🇹🇼🇾]?|🇶🇦?|🇷[🇪🇴🇸🇺🇼]?|🇸[🇦-🇪🇬-🇴🇷-🇹🇻🇽-🇿]?|🇹[🇦🇨-🇩🇫-🇭🇯-🇴🇷🇹🇻-🇼🇿]?|🇺[🇦🇬🇲🇸🇾-🇿]?|🇻[🇦🇨🇪🇬🇮🇳🇺]?|🇼[🇫🇸]?|🇽🇰?|🇾[🇪🇹]?|🇿[🇦🇲🇼]?|🎅[🏻-🏿]?|🏂[🏻-🏿]?|🏃[🏻-🏿]?|🏄[🏻-🏿]?|🏇[🏻-🏿]?|🏊[🏻-🏿]?|🏋[🏻-🏿]?|👁(‍🗨)?|👂[🏻-🏿]?|👃[🏻-🏿]?|👆[🏻-🏿]?|👇[🏻-🏿]?|👈[🏻-🏿]?|👉[🏻-🏿]?|👊[🏻-🏿]?|👋[🏻-🏿]?|👌[🏻-🏿]?|👍[🏻-🏿]?|👎[🏻-🏿]?|👏[🏻-🏿]?|👐[🏻-🏿]?|👦[🏻-🏿]?|👧[🏻-🏿]?|👨([🏻-🏿]|‍(❤(️(‍(👨|💋(‍👨))))|👨(‍(👦(‍👦)?|👧(‍[👦-👧])?))|👩(‍(👦(‍👦)|👧(‍[👦-👧])?))))?|👩([🏻-🏿]|‍(❤(️(‍(👩|💋(‍👩))))|👩(‍(👦(‍👦)?|👧(‍[👦-👧])?))))?|👮[🏻-🏿]?|👯[🏻-🏿]?|👰[🏻-🏿]?|👱[🏻-🏿]?|👲[🏻-🏿]?|👳[🏻-🏿]?|👴[🏻-🏿]?|👵[🏻-🏿]?|👶[🏻-🏿]?|👷[🏻-🏿]?|👸[🏻-🏿]?|👼[🏻-🏿]?|💁[🏻-🏿]?|💂[🏻-🏿]?|💃[🏻-🏿]?|💅[🏻-🏿]?|💆[🏻-🏿]?|💇[🏻-🏿]?|💪[🏻-🏿]?|🕵[🏻-🏿]?|🖐[🏻-🏿]?|🖕[🏻-🏿]?|🖖[🏻-🏿]?|🙅[🏻-🏿]?|🙆[🏻-🏿]?|🙇[🏻-🏿]?|🙋[🏻-🏿]?|🙌[🏻-🏿]?|🙍[🏻-🏿]?|🙎[🏻-🏿]?|🙏[🏻-🏿]?|🚣[🏻-🏿]?|🚴[🏻-🏿]?|🚵[🏻-🏿]?|🚶[🏻-🏿]?|🛀[🏻-🏿]?|🤘[🏻-🏿]?";
```

Comment:
Yep, agree with this and we'll definitely want to update it from time to time as new emoji are accepted.